camoco/COB.py

```diff
@@ -1499,6 +1503,8 @@ def _calculate_gene_hierarchy(self,method='single'):
         dists = (dists * pcc_std) + pcc_mean
         dists = np.tanh(dists)
         dists = 1 - dists
+        #convert nan to 0's, linkage can only use finite values
```

Comment:
Excellent PR though, this fixes that linkage bug thats been killing our CI test for a while.